recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
+    - versioneer
```

Comment:
Maybe move to host? (not fully understanding the difference)

Reply:
was just about to try that :)

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host 
So I think if both build and host are defined, what we have in pyproject.toml `[build-system]`'s "requires" field is basically what corresponds to "host" (since in pyproject.toml you don't specify compilers)